subprojects/packagefiles/utf8proc/meson.build

```diff
@@ -31,3 +34,58 @@ pkg.generate(
 )
 
 install_headers('utf8proc.h')
+
+curl = find_program('curl', required: get_option('tests'))
+
+if get_option('tests').require(curl.found()).allowed()
```

Comment:
this looks like a more complicated version  of

if curl.found()

Reply:
The goal is to be able to use `--auto-features=disabled`, as always.